{
  "id": "CVE-2025-57727",
  "sourceIdentifier": "cve@jetbrains.com",
  "published": "2025-08-20T10:15:30.043",
  "lastModified": "2025-08-20T14:39:07.860",
  "vulnStatus": "Undergoing Analysis",
  "descriptions": [
    {
      "lang": "en",
      "value": "In JetBrains IntelliJ IDEA before 2025.2 credentials disclosure was possible via remote reference"
    },
    {
      "lang": "es",
      "value": "En JetBrains IntelliJ IDEA antes de 2025.2 la divulgaci√≥n de credenciales era posible mediante referencia remota"
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "cve@jetbrains.com",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:N/A:N",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "NONE",
          "userInteraction": "REQUIRED",
          "scope": "CHANGED",
          "confidentialityImpact": "LOW",
          "integrityImpact": "NONE",
          "availabilityImpact": "NONE",
          "baseScore": 4.7,
          "baseSeverity": "MEDIUM"
        },
        "exploitabilityScore": 2.8,
        "impactScore": 1.4
      }
    ]
  },
  "weaknesses": [
    {
      "source": "cve@jetbrains.com",
      "type": "Primary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-319"
        }
      ]
    }
  ],
  "references": [
    {
      "url": "https://www.jetbrains.com/privacy-security/issues-fixed/",
      "source": "cve@jetbrains.com"
    }
  ]
}